spec/ddtrace/contrib/grpc/support/grpc_helper.rb

```diff
@@ -0,0 +1,88 @@
+require 'grpc'
+
+module GRPCHelper
```

Comment:
Not a huge deal, but food for thought: consider turning this into an RSpec shared context that you can compose into the other example files.

Then you can stub the constants to make them more composable. A reason for doing so is that it allows the RSpec example to optionally override and tweak behavior, without having to define constants per example.

e.g. a brief example of this could look like:

```ruby
# Shared context
RSpec.shared_context 'GRPC service' do
  def run_request_reply(address = '0.0.0.0:50052', client = nil)
    runner(address, client) { |c| c.basic(message_class.new) }
  end

  # Stubbed constant
  # This message class can be overidden in an example to change behavior
  let(:message_class) do
    stub_const('TestMessage', Class.new do
      class << self
         def marshal(_o)
           ''
         end
   
         def unmarshal(_o)
           new
         end
       end
    end)
  end
end

# Compose it into a test
RSpec.describe "GRPC test" do
  include_context 'GRPC service'
end
```

For a better example in practice, check out the `Resque job` shared context here: https://github.com/DataDog/dd-trace-rb/pull/405/files

This is only a suggestion though; only implement it if you think it will benefit these tests.